src/server/controllers/pokemon.js

```diff
@@ -64,8 +64,18 @@ const getPokemons = async(req, res) => {
     res.json({ data: pokemons })
 }
 
+const getPokemonRatings = async (req, res) => {
+    const pokeRatings = await prisma.pokemon.findMany({
+        include: {
+            rating: true
+        }
+    })
+    res.json({pokemonRatingData: pokeRatings})
```

Comment:
Shouldn't this conform to the same format that the other routes in your API have? `{ data: pokeRatings }`